package.json

```diff
@@ -18,6 +20,7 @@
         "crypto-browserify": "^3.12.0",
         "csv-parser": "^3.0.0",
         "detect-browser": "^5.3.0",
+        "ethers": "^5.7.1",
```

Comment:
Same question, why do we need this dependency ?

Reply:
Same as above.